postman/collections/Interledger.json

```diff
@@ -1654,12 +1620,10 @@
 									}
 								],
 								"url": {
-									"raw": "http://localhost:4000/incoming-payments/{{incomingPaymentId}}",
-									"protocol": "http",
+									"raw": "{{pfryWalletAddress}}/incoming-payments/{{incomingPaymentId}}",
```

Comment:
I think the postman collection needs to be updated again - looks like it swallowed the "remove wallet address from url" changes.